### ISSUE TITLE
addGroupedLayersControl(): baseGroups should accept list of vectors as overlayGroups does (feature request)

### ISSUE THREAD

**Issue by aloboa** [trafficonese/leaflet.extras#228]
See https://github.com/r-spatial/leafem/issues/91

As far as I understand the doc and example of addGroupedLayersControl()
https://trafficonese.github.io/leaflet.extras/reference/addGroupedLayersControl.html ,
only overlayGroups accepts a list of group vectors, thus only overlayGroups can be organized.
In my case, where I need to introduce structure is for base groups:
https://diapiro.geo3bcn.csic.es/alobo/blog/IcelandCampaign2023.html
as I have base layers for the general region and then some layers for specific sub-regions.
Also, in other cases, grouping base layers by type would also make sense.
Could you consider a request feature to accept a list of named group vectors for baseGroups as for overlayGroups ?

**Comment by trafficonese** [comment]
unfortunately the plugin is only for overlay groups..

**Comment by aloboa** [comment]
> unfortunately the plugin is only for overlay groups..

Yes, I understand. My feature request is extending it to base groups. Please consider it.
